### ISSUE TITLE
Attachments wrongly assigned

### ISSUE THREAD

**Issue by wattazoum** [bsorrentino/maven-confluence-plugin#116]
When I setup the site.xml to put attachments to the Home page, they are put under the parent
```xml
  <home uri="../markdown/index.md">

    <attachment name="support.png" uri="../resources/support.png"/>
    <attachment name="questions.png" uri="../resources/questions.png"/>
    <attachment name="contribute.png" uri="../resources/contribute.png"/>
```
The files are not put under the index page, but to the page upper

**Comment by wattazoum** [comment]
In fact, in case of plugin site generation,  even the labels are put to the parentPage and not the home.

**Comment by bsorrentino** [comment]
ix released in release `4.12`
